pkg/gardener/shoot/extender/provider/provider.go

```diff
@@ -218,6 +219,10 @@ func getConfig(runtimeShoot imv1.RuntimeShoot, zones []string, enableDualStack b
 		{
 			return getConfigForProvider(runtimeShoot, openstack.GetInfrastructureConfig, openstack.GetControlPlaneConfig)
 		}
+	case hyperscaler.TypeAlicloud:
+		{
+			return getConfigForProvider(runtimeShoot, alicloud.GetInfrastructureConfig, alicloud.GetControlPlaneConfig)
```

Comment:
There are missing unit tests specific for alicloud provider in file: 

`provider_alicloud_test.go`